src/libsyntax/print/pprust.rs

```diff
@@ -3186,10 +3186,11 @@ mod tests {
     a. ..;
     for token in program.chars()
     {
-        match token {
-                        '+' => accumulator += 1 , '-' => accumulator -= 1 , '*' => accumulator *= 2 , '/' => accumulator /= 2 , _ => {}
+        match token 
+        {
+            \'+\' => accumulator += 1 , \'-\' => accumulator -= 1 , \'*\' => accumulator *= 2 , \'/\' => accumulator /= 2 , _ => {}
```

Comment:
I dont really understand print::pp to be honest. Documentation would be really appreciated.